samples/todolist-aspnetcore/.gitignore

```diff
@@ -0,0 +1,288 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
We shouldn't need this .gitignore, and if we are missing any of the important ignore statements, lets add it to the one at the root of the repository.